app/App.jsx

```diff
@@ -590,6 +596,10 @@ class App extends React.Component {
                                     path="/instant-trade/:marketID"
                                     component={QuickTrade}
                                 />
+		                <Route
+		                    path="/poolmart"
```

Comment:
what is poolmart?

Reply:
Poolmart is a uniqie term that was defined by me to call the "liquidity pool markets" page of BitShares, perhaps you suggest what to call that page since we have a page called "Exchange" already and we shouldn't mix between Pools Market and Exchange.